src/lex.js

```diff
@@ -61,6 +59,31 @@ function asyncTrigger() {
   };
 }
 
+function fromCodePoint(codepoint) {
+  if (String.fromCodePoint) {
+    return String.fromCodePoint(codepoint);
```

Comment:
(Also line 77)

These lines are reported as not tested by coveralls (https://coveralls.io/builds/2635416/source?filename=src%2Flex.js#L64), because they never run on node (but `String.fromCodePoint` and `String.prototype.codePointAt` are supportd by iojs)
Is this a problem?